habitat/sims/habitat_simulator.py

```diff
@@ -124,10 +126,12 @@ def get_observation(self, sim_obs):
 
 
 SIM_ACTION_TO_NAME = {
-    0: SimulatorActions.FORWARD.value,
-    1: SimulatorActions.LEFT.value,
-    2: SimulatorActions.RIGHT.value,
-    3: SimulatorActions.STOP.value,
+    0: SimulatorActions.STOP.value,
```

Comment:
Changing the indices of actions should wait until after the challenge IMO.  We don't want people to have to deal with that confusion especially since we overwrite their install of HAPI on the eval server.

If this effects the release of SplitNet code, you can monkey patch HAPI to have this ordering in the SplitNet code.